src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataFetcher.java

```diff
@@ -101,7 +101,7 @@
   /**
    * A cache of local repositories.
    */
-  final LRUMap<GitHubProject, LocalRepository> localRepositories
+  static final LRUMap<GitHubProject, LocalRepository> localRepositories
```

Comment:
If you'd like to make it static, please follow the naming conventions for static field. It should be `LOCAL_REPOSITORIES`. The static field should be moved to the other static fields.